File: 30 Days of Code/04 - Day 3 - Intro to Conditional Statements.py
# ========================
#       Information
# ========================

# Direct Link: https://www.hackerrank.com/challenges/30-conditional-statements/problem
# Difficulty: Easy
# Max Score: 30
# Language: Python

# ========================
#         Solution
# ========================

if __name__ == '__main__':
    N = int(input())

    if N%2 != 0 or ((N > 5 and N < 21) and N%2 == 0):
        print("Weird")
    else:
        print("Not Weird")

File: 30 Days of Code/C#/04 - Day 3 - Intro to Conditional Statements.cs
// ========================
//       Information
// ========================

// Direct Link: https://www.hackerrank.com/challenges/30-conditional-statements/problem
// Difficulty: Easy
// Max Score: 30
// Language: C#

// ========================
//         Solution
// ========================

using System;

class Solution {
    static void Main(String[] args) {
        var n = int.Parse(Console.ReadLine());

        if (n % 2 != 0) {
            Console.WriteLine("Weird");
        }
        else {
            if (n <= 5) { 
                Console.WriteLine("Not Weird");
            }
            else if (n <= 20) {
                Console.WriteLine("Weird");
            }
            else {
                Console.WriteLine("Not Weird");
            }
        }
    }
}
